testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -48,11 +48,6 @@ Feature: Be able to list available channels and enable them
     And I should see "Total time:" in the output
     And I should see "Repo URL:" in the output
 
-@susemanager
-  Scenario: Enable SLES15 SP4 SUMA client tools for creating bootstrap repositories
```

Comment:
we write elsewhere `SLES 15 SP4`

Reply:
Ok, but as you can see this is code we are removing.

Reply:
"oops"